codeserver/ubi9-python-3.9/Dockerfile

```diff
@@ -63,10 +63,12 @@ RUN yum install -y https://download.fedoraproject.org/pub/epel/epel-release-late
     INSTALL_PKGS="bind-utils nginx nginx-mod-stream nginx-mod-http-perl fcgiwrap initscripts chkconfig" && \
     yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
     rpm -V $INSTALL_PKGS && \
-    # spawn-fcgi is not in epel9 \
-    rpm -i --nodocs https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/37/Everything/x86_64/os/Packages/s/spawn-fcgi-1.6.3-23.fc37.x86_64.rpm && \
+    # Install supervisord
+    yum -y install supervisor && \
```

Comment:
Why is this kept as a separate yum transaction rather going directly to the `INSTALL_PKGS` above?

Reply:
Good catch, thanks for the review! I will do that now, it is true what Jiri mentioned on the bellow comment, however, i can not find the track issue either.